pandora-common/src/utility/cards.ts

```diff
@@ -0,0 +1,183 @@
+import { z } from 'zod';
+import type { CharacterId } from '../character/characterTypes.ts';
+
+// Define Suits and Ranks
+const suits = [
+	'\u2661', //hearts
+	'\u2662', //diamonds
+	'\u2667', //clubs
+	'\u2664', //spades
+] as const;
+const ranks = ['2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K', 'A'] as const;
+
+// Define CardGameCard class
+export class CardGameCard {
+	public readonly suit: string;
+	public readonly rank: string;
```

Comment:
If you update the schemas like this:
```ts
const SuitSchema = z.enum(suits);
type CardGameSuit = z.infer<typeof SuitSchema>;

const RankSchema = z.enum(ranks);
type CardGameRank = z.infer<typeof RankSchema>;
```

Then you can update the card to be:
```ts
public readonly suit: CardGameSuit;
public readonly rank: CardGameRank;
```
Which will make it easier to spot mistakes in the future.